app/views/result/index.html.erb

```diff
@@ -48,8 +57,7 @@
         the right format                                  %>
     <% url = 'https://www.gov.uk/api/search.json?q=' %>
     <% term = params["search_term"] %>
-    <% url += term.split.join("+") %>
-
+    <%  url += term.split.join("+") %>
```

Comment:
While I would recommend not declaring variables in views if you are going to do it you can just do the following:
```
<% url = 'https://www.gov.uk/api/search.json?q=' 
   term = params["search_term"] 
   url += term.split.join("+") %>